src/cloudflare/internal/vectorize-api.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2023 Cloudflare, Inc.
 // Licensed under the Apache 2.0 license found in the LICENSE file or at:
 //     https://opensource.org/licenses/Apache-2.0
-import * as flags from 'workerd:compatibility-flags'
+import { default as flags } from "workerd:compatibility-flags";
```

Comment:
Couldn't this just be the following?
```suggestion
import flags from "workerd:compatibility-flags";
```

Reply:
Yes, it can. I honestly prefer the `{ default as flags }` because it is more explicit but it's good either way.

Reply:
That's fair - just thought I should point it out, as importing it with a name tends to be more common than destructuring the property, but like you said, doesn't make a big difference 🙂